src/network/graphRequest.js

```diff
@@ -0,0 +1,106 @@
+import _ from 'lodash';
+import gql from 'graphql-tag';
+import { ApolloClient } from 'apollo-client';
+import { HttpLink } from 'apollo-link-http';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { endpoint } from '../config/app';
+import GraphParser from './graphParser';
+import { isValidEnum, getQueryFields } from './graphDataStruct';
+
+const client = new ApolloClient({
+  link: new HttpLink({ uri: endpoint.graphql }),
+  cache: new InMemoryCache(),
+});
+
+class GraphRequest {
+  constructor(queryName) {
+    this.queryName = queryName;
+    this.filters = {};
+  }
+
+  setFilters(filters) {
+    this.filters = filters;
+  }
+
+  build() {
+    let query = `query {${this.queryName}`;
+
+    if (_.isEmpty(this.filters)) {
+      query = query.concat('{');
+    } else {
+      // Create entire string for OR: [] as objects
+      let filterStr = '';
+      _.forEach(this.filters, (obj, index) => {
+        if (!_.isEmpty(filterStr)) {
+          filterStr = filterStr.concat(',');
+        }
+        const str = Object
+          .keys(obj)
+          .map((key) => {
+            if (isValidEnum(key, obj[key])) {
+              // Enums require values without quotes
+              return `${key}: ${obj[key]}`;
```

Comment:
Suggestion: all keys in grapghql query doesnt require quotes, only string value require it. So we should add quotes when value is string type

Reply:
done